### ISSUE TITLE
Graphics glitch

### ISSUE THREAD

**Issue by pasha002** [ValveSoftware/Dota2-Gameplay#11461]
### Description

Whenever I play online graphics glitch out like so, this isnt a problem when i play with bots. I cant tell if its an issue with the game or an hard ware issue .

Macbook air
ios 12.6

![Screenshot 2023-09-02 at 23 25 40](https://github.com/ValveSoftware/Dota2-Gameplay/assets/143958514/1bda3ca3-a474-46b8-95f7-1f82c3e3723e)

### Screenshots

![Screenshot 2023-09-02 at 23 25 40](https://github.com/ValveSoftware/Dota2-Gameplay/assets/143958514/1bda3ca3-a474-46b8-95f7-1f82c3e3723e)

**Comment by VOLBAX2** [comment]
Mac Book pro (Retina, 2012) have this problem too, 5 days can't play game. New update was broked life for all Mac player's,...Valve don't do anything, they have smurfs, they 🙈 (see)All, but didn't see biggest one

**Comment by pasha002** [comment]
rip bro im sorry to hear that like 3 days ago its fine now its not

**Comment by phat347** [comment]
valve need to fix this as soon as possible. I also have this map bug and abandon 2 games in a row
<img width="1598" alt="Screen Shot 2023-09-02 at 1 32 13 PM" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/39620179/796e9115-f9b6-4466-9172-938b0393ce0a">

<img width="1018" alt="Screen Shot 2023-09-01 at 7 30 13 PM" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/39620179/7cc698b6-ad4b-4f1e-9d09-6e5033995d2b">

**Comment by memobick** [comment]
i have the same bug. macbook pro intel machine

**Comment by shaileshgarg** [comment]
https://github.com/ValveSoftware/Dota2-Gameplay/issues/11199#issuecomment-1702255860. try this ?

**Comment by memobick** [comment]
> https://github.com/ValveSoftware/Dota2-Gameplay/issues/11199#issuecomment-1702255860. try this ?

yes i did. after posting my comment. worked like a charm! blessed be our lord and saviour gaben
